lib/mixins/dropdown.js

```diff
@@ -81,6 +80,9 @@ export default {
                         el.addEventListener('mouseover', this.noop);
                     });
                 }
+
+                document.addEventListener('keydown', this.keydownListener);
```

Comment:
Also, since we to focus management, the indexes of the drop down items should remain at -1